### DISCUSSION TITLE
Flink CDC 2.0将数据倒入到mysql或oracle或sqlserver或postgres，有什么简单的方法吗？

### DISCUSSION THREAD

**Discussion by harmonyww** [apache/flink-cdc#1228] [Q&A]
Flink CDC 2.0将数据倒入到mysql或oracle或sqlserver或postgres，有什么简单的方法吗，

Flink SQL>  INSERT INTO products_mys
>  SELECT p.ID, p.NAME, p.DESCRIPTION
>  FROM products_ora AS p;
[INFO] Submitting SQL update statement to the cluster...
[ERROR] Could not execute SQL statement. Reason:
org.apache.flink.table.api.ValidationException: Connector 'mysql-cdc' can only be used as a source. It cannot be used as a sink.

烦请给一个例子的完整配置，从oracle到数据到mysql，怎么配置？

**Comment by linqichen2020** [comment]
这个问题不难，写3个sql。 第一个sql是建源表t0，connector是 mysql-cdc.  第二个是建输出表t1，connector是jdbc, 第三个是 insert .  类似于 insert into t1 selct xx from t0;

**Comment by dk333** [comment]
题主 sink和source都没搞清楚，对应connector也不知道
